demo/App.jsx

```diff
@@ -1,11 +1,62 @@
-import React from 'react';
+import React, {useState} from 'react';
 import ReactDOM from 'react-dom';
-import {WebexMeetingWidget} from '../src';
+import {Input, Sidebar, SidebarBody, SidebarHeader, SidebarNav, SidebarNavItem} from '@momentum-ui/react';
+
+import WebexMeetingWidgetDemo from './WebexMeetingWidgetDemo';
 
 import './App.css';
 
 export default function App() {
-  return <WebexMeetingWidget accessToken="<YOUR_ACCESS_TOKEN>" meetingDestination="MEETING_DESTINATION" />;
+  const [tokenInput, setTokenInput] = useState('');
+  const [token, setToken] = useState();
+
+  const clearToken = () => {
+    setTokenInput('');
+    setToken();
+  };
+
+  return (
+    <div className="main">
+      <Sidebar withIcons={false}>
+        <SidebarBody>
+          <SidebarNav title="Webex Widgets">
+            <SidebarNavItem title="WebexMeeting" />
+          </SidebarNav>
+        </SidebarBody>
+      </Sidebar>
+      <div className="content">
+        <div className="section">
```

Comment:
There is a semantic HTML element, [`section`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section). I think this also represents the intention in a semantic way 😬